README.md

```diff
@@ -494,6 +494,22 @@ This is a [link to an existing confluence page](ac:Pagetitle)
 And this is how to link when the linktext is the same as the [Pagetitle](ac:)
 ```
 
+### Add width for an image
+
+Use the following macro:
+```markdown
+<!-- Macro: \!\[.*\]\((.+)\)\<\!\-\- (.*) \-\-\>
+     Template: ac:image
+     Attachment: ${1}
+     Width: ${2} -->
+```
+And attach any image with the following
+```markdown
+![Example](../images/example.png)<!-- 300 -->
```

Comment:
Nice idea!

I'd make it more explicit, i.e. `<!-- width=300 -->` so in the future we will be able to add `height=` as well, but I it's ok to leave it like this for the sake of example

Reply:
Yes, that sounds great. I allowed the maintainers to add new commits to my branch so if you'd like, please add it as a commit.
` <!-- Macro: \!\[.*\]\((.+)\)\<\!\-\- (.*) \-\-\> ` turns to ` <!-- Macro: \!\[.*\]\((.+)\)\<\!\-\- width=(.*) \-\-\> ` and `![Example](../images/example.png)<!-- 300 -->` turns to `![Example](../images/example.png)<!-- width=300 -->`.

Reply:
Added.